### PR TITLE
Fixed some bugs

### LINKED ISSUE #31: Can't create event via admin panel
Log output
Started POST "/admin/events" for 127.0.0.1 at 2016-02-22 10:48:45 +0200
Processing by Admin::EventsController#create as HTML
  Parameters: {"utf8"=>"✓", "authenticity_token"=>"/jUFsczyEh2+hrfjN/3Gz35/Y1tE3oekApBhRXMb1KxSrWkebHA2ZmmgQrYCHK22+g7kUWOuhVjYQxis9Kgidg==", "event"=>{"subject"=>"friendship is magic", "datetime"=>"2016-03-10 13:00", "location_id"=>"1", "confirmed"=>"1", "user_ids"=>["", "1"]}, "commit"=>"Create Event"}
  User Load (0.3ms)  SELECT  "users".\* FROM "users" WHERE "users"."id" = $1  ORDER BY "users"."id" ASC LIMIT 1  [["id", 1]]
  Event Load (0.2ms)  SELECT  "events".\* FROM "events" WHERE "events"."id" = $1 LIMIT 1  [["id", nil]]
Completed 404 Not Found in 5ms (ActiveRecord: 0.5ms)

ActiveRecord::RecordNotFound - Couldn't find Event with 'id'=:
  activerecord (4.2.4) lib/active_record/core.rb:155:in `find'
  app/admin/event.rb:8:in`set_users'

### LINKED ISSUE #41: Datepicker doesn't works in Safary
This bug is minor. Leave it right now

### PR DESCRIPTION
**Pull request by sparksofl**
Close #31, #41, #42

### CODE REVIEW

File: app/admin/event.rb [hunk 1]
```diff
@@ -1,20 +1,16 @@
 ActiveAdmin.register Event do
   permit_params :subject, :datetime, :confirmed, :location_id,
-                :user_events, :users
-  before_action :set_users_before_update, only: [:update]
+                :user_events, :users, user_ids: []
+  before_action :add_users_to_event, only: [:update]
```
Comment (brytannia) [inline app/admin/event.rb]: these parentheses aren't obligatory. you can remove them

Comment (sparksofl) [inline app/admin/event.rb]: user_ids: []
[] these?

File: app/admin/event.rb [hunk 2]
```diff
@@ -58,7 +54,7 @@ def set_users_before_update
       f.input :confirmed
 
       f.input :users,
-              collection: User.all.map { |user| [user.email, user.id] },
+              collection: User.all.map { |user| [user.first_name + ' ' + user.last_name, user.id] },
```
Comment (brytannia) [inline app/admin/event.rb]: user.first_name + ' ' + user.last_name is a string. better to use string with double quotes

File: app/admin/event.rb (hunk 1 above)
Comment (brytannia) [inline app/admin/event.rb]: only: :update

**Comment by brytannia** [comment]
:+1:

**Pull request merged**